web/tests/pm_list_data.test.js

```diff
@@ -111,7 +111,7 @@ test("get_conversations", ({override}) => {
             },
         },
         {
-            recipients: "Alice, Bob",
+            recipients: "Alice and Bob",
```

Comment:
So I think we probably want the `short` / comma-separated format for the left sidebar at the very least, since there's not very much space there and we'd rather use it for actual content.

This is probably a case where we will want to revisit the decision after completing projects like #27146 that make space less of a constraint.

Reply:
If we want to just stick to short / comma-separated format in left sidebar I have to refactor ```people.ts``` ```get_recipients``` function. And make it return an array of names, and then join them using ```util.format_array_as_list``` to the desired format.

Do we want to use ```short``` format: 
![image](https://github.com/zulip/zulip/assets/91906953/5b6884e0-f5a3-4bd7-bdec-1d853fce557d)

Or use "narrow" format: 
![image](https://github.com/zulip/zulip/assets/91906953/2ea91ad3-132e-415f-9a6b-45a8c70c0b7e)

Reply:
As of now I have used ```Short``` format here.

Reply:
@alya thoughts on which format you prefer?

Reply:
Ah, sorry I missed the question earlier. `narrow` seems fine. We don't use `&` elsewhere, and I don't think it's an improvement.

Reply:
Done.